src/Identity/IdentityServer/DeviceValidator.cs

```diff
@@ -66,10 +78,16 @@ await _mailService.SendNewDeviceLoggedInEmail(user.Email, deviceType, now,
         return null;
     }
 
+    /// <summary>
+    /// Check if a device is known to the user.
+    /// </summary>
+    /// <param name="user">current user trying to authenticate</param>
+    /// <param name="request">contains raw information that is parsed about the device</param>
+    /// <returns>true if the device is known, false if it is not</returns>
```

Comment:
If we have the summary on the interface, I don't think we need it on the implementation.